styles.css

```diff
@@ -0,0 +1,219 @@
+/* ==============
+   UTILITY STYLES
+   ============== */
+
+/* Corrects the layout around floated elements */
+.u-clearfix::before,
+.u-clearfix::after {
+  content: " ";
+  display: table;
+}
+
+.u-clearfix::after {
+  clear: both;
+}
+
+/* Displays list items as inline block  */
+.u-list-inline li {
+  display: inline-block;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+
+body {
+  background-color: #FCFCFC;
+  color: var(--color-slate);
+  font-family: 'Crimson Text', serif;
+  font-size: 9px;
+  line-height: 1.3;
+  margin: 0;
+}
+
+.donut-img {
+  position: relative;
+  top: -20px;
+  left: -75px;
+}
+
+a {
+  color: rgb(29, 38, 98);
+}
+
+h2 {
+  color: rgb(29, 38, 98);
+  position: absolute;
+  top: 25px;
+  right: 15px;
+  font-size: 20px;
+  font-family: 'Arvo', serif;
+}
+
+span {
+  color: rgb(255, 50, 0);
+  font-family: 'Dancing Script', cursive;
+  font-size: 25px;
+  position: absolute;
+  top: -22px;
+}
+
+
+.spicerack {
+  width: 60px;
+  position: absolute;
+  top: 20px;
+  right: 107px;
+}
+
+
+.content-default {
+  color: rgb(255, 252, 222);
+  width: 148px;
+  text-align: left;
+  position: absolute;
+  top: 72px;
+  left: 315px;
+}
+
+.content-alt {
+  color: rgb(255, 252, 222);
+  width: 148px;
+  text-align: left;
+  position: absolute;
+  top: 72px;
+  left: 315px;
+}
+
+.trigger {
+  width: 30px;
+}
+
+.menu {
+  list-style-type: none;
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  font-size: 12px;
+}
+
+.cooktime {
+  position: absolute;
+  top: 230px;
+  left: 25px;
+}
+
+.servings {
+  position: absolute;
+  top: 219px;
+  left: 100px;
+}
+
+.ingredientswhole {
+  position: absolute;
+  top: 234px;
+  left: 200px;
+}
```

Comment:
I think you can see the issue with absolute positioning here -- it's a lot of effort to get these things lined up pixel perfect and nicely in line.